x/gamm/types/msgs_test.go

```diff
@@ -867,3 +872,148 @@ func TestMsgExitSwapShareAmountIn(t *testing.T) {
 		}
 	}
 }
+
+// Test authz
```

Comment:
Please elaborate elaborate here that this test is meant more so to test the authz than the gamm message